src/setup.py

```diff
@@ -24,6 +21,49 @@
 # 5 - Success message (or failed)
 # 6 - ask the user delete the folder and exit
 
+# Verifying if all required files are included in /dist/
+try:
+
+    print("Verifying required files...")
+
+    source = os.path.join(os.path.abspath(os.curdir), "discordRPC\\dist")
+
+    # If any files are needed within the /dist/ folder, add them here
+    requiredFiles = ("checkupdate.exe", "hoi4Presence.exe", "version.json", "runRPC.bat")
+    filesNotFound = []
+
+    # Checking if the required files are in /dist/
+    with os.listdir(source) as distContents:
```

Comment:
I am getting an error and it seems to be in this line of the code:

![image](https://github.com/ThiaudioTT/hoi4-presence/assets/43919529/c684f6c8-53f2-4e1c-b064-aed52cc367a9)